### ISSUE TITLE
SDL_SetWindowIcon not implemented

### ISSUE THREAD

**Issue by NoxiousPluK** [GabrielFrigo4/SDL-Sharp#46]
I would like to set an icon for my window, but it seems that this is not yet implemented:
https://wiki.libsdl.org/SDL2/SDL_SetWindowIcon

Using SDL2-CS I would do something like:
```
IntPtr ico = SDL_image.IMG_Load("Resources\\Icon.png");
SDL.SDL_SetWindowIcon(Config.Window, ico);
SDL.SDL_FreeSurface(ico);
```

**Comment by GabrielFrigo4** [comment]
> I would like to set an icon for my window, but it seems that this is not yet implemented: https://wiki.libsdl.org/SDL2/SDL_SetWindowIcon
> 
> Using SDL2-CS I would do something like:
> 
> ```
> IntPtr ico = SDL_image.IMG_Load("Resources\\Icon.png");
> SDL.SDL_SetWindowIcon(Config.Window, ico);
> SDL.SDL_FreeSurface(ico);
> ```

I made a Commit, test it to see if it's working

**Comment by NoxiousPluK** [comment]
Had to make a small fix, see #47

But, this works great now:
```
IMG.Load("Resources\\logo.png", out PSurface icon);
SDL.SetWindowIcon(mainWindow, icon);
```

Thanks a lot!
